### DISCUSSION TITLE
Please upgrade to @mapbox/node-pre-gyp warning

### DISCUSSION THREAD

**Discussion by guidoslotboom** [nuxt/nuxt#17821] [Questions] [answered]
How can I avoid the following warning?

```
warning nuxt3 > @nuxt/nitro > @vercel/nft > node-pre-gyp@0.13.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
```

I've done the following upgrade lines, but I keep getting the warning.

```
npm uninstall node-pre-gyp --save
npm install @mapbox/node-pre-gyp --save
```

What am I doing wrong or is this warning still unavoidable? Is there anyone who can answer this for me?

**Comment by danielroe** [comment, marked as answer]
This is a consequence of https://github.com/vercel/nft, who have _both_ packages in their dependency tree. I believe it's for backward-compatibility: https://github.com/vercel/nft/pull/206#discussion_r631837091. But it's not something you need to worry about.

https://github.com/vercel/nft/blob/main/package.json#L21-L33

**Reply by guidoslotboom** [reply]
Thanks for your response and good to know I can ignore the warning.
